exchanges/huobi/huobi.go

```diff
@@ -877,6 +884,10 @@ func (h *HUOBI) SendAuthenticatedHTTPRequest(ep exchange.URL, method, endpoint s
 		Verbose:       h.Verbose,
 		HTTPDebugging: h.HTTPDebugging,
 		HTTPRecording: h.HTTPRecording,
+	}
+
+	err = h.SendPayload(ctx, request.Unset, func() (*request.Item, error) {
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. Huobi has rate limits